alias.bash

```diff
@@ -0,0 +1,220 @@
+# vim: set filetype=sh:
+echo "sourcing ${PWD}/alias.bash ..."
+
```

Comment:
We really shouldn't be storing these alias files in git, as every user should have their own custom alias functions kept wherever they feel is appropriate. We shouldn't force anyone to use a certain set of aliased functions that we define for them. This could also cause problems in complex environments, where users could have conflicting alias names.

Optimally, I would suggest removing this function from the repo and not placing it under version control any longer.

Reply:
good idea, I think it is better to be stored somewhere like infiles which is user specific